src/main/java/Ixo.java

```diff
@@ -0,0 +1,151 @@
+import java.util.Scanner;
+
+
+public class Ixo {
+
+    public final static String SEPARATOR = "____________________________________________________";
+
+    public static void menu() {
+        System.out.println("1. Task List");
+    }
+
+    public static void taskList() {
+
+        Scanner lineScan;
+        Task[] taskStore = new Task[100]; //limit list to store only 100 items
```

Comment:
Try and avoid the usage of magic literals by storing them in variables or constants. Something like `MAX_TASKS` or something would be better.